docs/runbook/Features.md

```diff
@@ -246,10 +246,13 @@ POPs to the network or reducing the requested demand is a better alternative.
 To use the Maximize Common Bandwidth feature, set `MAXIMIZE_COMMON_BANDWIDTH` under
 the `NETWORK_DESIGN` section to True.
 
-## Extend Existing Candidate Graph
+## Extend Base Topology
 
-This feature enables extending an already deployed "base network" with new
-sites. LOS will be computed between the base network sites and the new sites
+This feature enables extending a base topology with new sites, where a base
```

Comment:
Maybe: This feature enables the addition of new sites to a base topology with known sites and links. The sites...

LOS will be computed between the sites in the base topology and the new sites and between each of the new sites. The new candidate topology includes the base topology as a subgraph. The candidate topology is then optimized to generate a network plan.